topics/dev/tutorials/conda/slides.html

```diff
@@ -0,0 +1,644 @@
+         -U        convert all bases to uppercases
+         -S        strip of white spaces in sequences
+(seqtk_seq) $ conda_env_deactivate
+$
+```
+
+---
+
+### Using our Tool with the Conda Environment
+
+Now that we have verified the Conda environment setup with `conda_install` works properly on the 
+command-line, we can use our tool!
+
+`planemo test` and `planemo serve` will use this environment by default now for this tool.
+
+---
+
+### Planemo `test`
+
+```
+$ planemo test seqtk_seq.xml
+...
+INFO  [galaxy.tools.actions] Handled output named output1 for tool seqtk_seq (20.136 ms)
+INFO  [galaxy.tools.actions] Added output datasets to history (12.782 ms)
+INFO  [galaxy.tools.actions] Verified access to datasets for Job[unflushed,tool_id=seqtk_seq] (10.954 ms)
+INFO  [galaxy.tools.actions] Setup for job Job[unflushed,tool_id=seqtk_seq] complete, ready to flush (21.053 ms)
+INFO  [galaxy.tools.actions] Flushed transaction for job Job[id=2,tool_id=seqtk_seq] (26.510 ms)
+INFO  [galaxy.jobs.handler] (2) Job dispatched
+DEBUG [galaxy.tools.deps] Using dependency seqtk version 1.2 of type conda
+DEBUG [galaxy.tools.deps] Using dependency seqtk version 1.2 of type conda
+INFO  [galaxy.jobs.command_factory] Built script [/tmp/tmpLvKwta/job_working_directory/000/2/tool_script.sh] for tool command [[ "$CONDA_DEFAULT_ENV" = "/Users/john/miniconda2/envs/__seqtk@1.2" ] || . /Users/john/miniconda2/bin/activate '/Users/john/miniconda2/envs/__seqtk@1.2' >conda_activate.log 2>&1 ; seqtk seq -a '/tmp/tmpLvKwta/files/000/dataset_1.dat' > '/tmp/tmpLvKwta/files/000/dataset_2.dat']
+DEBUG [galaxy.tools.deps] Using dependency samtools version None of type conda
+DEBUG [galaxy.tools.deps] Using dependency samtools version None of type conda
+ok
+
+----------------------------------------------------------------------
+XML: /private/tmp/tmpLvKwta/xunit.xml
+----------------------------------------------------------------------
+Ran 1 test in 15.936s
+
+OK
+```
+
+The following line indicates the seqtk package was found:
+
+```
+[galaxy.tools.deps] Using dependency seqtk version 1.2 of type conda
+```
+
+---
+
+### <i class="fa fa-pencil" aria-hidden="true"></i> Hands-on
+
+![](../../images/exercise.png)
+
+---
+
+### <i class="fa fa-pencil" aria-hidden="true"></i> Hands-on
+
+#### The Goal
+- Use the Planemo commands `conda_install`, `conda_env`, and `test` to learn the Galaxy tool depenendency development lifecycle.
+
+---
+
+### <i class="fa fa-pencil" aria-hidden="true"></i> Hands-on
+
+#### Steps
+
+Run the following commands to practice working with Galaxy tools, Planemo, and Conda.
+
+1. `planemo project_init --template=seqtk_complete seqtk_example`
+2. `cd seqtk_example`
+3. `planemo conda_install seqtk_seq.xml`
+4. `planemo conda_env seqtk_seq.xml`
+5. `planemo test seqtk_seq.xml`
+
+---
+
+### Finding the Correct Requirements & Packages
+
+The above example worked because a published Bioconda recipe named`seqtk` at version `1.2`
```

Comment:
@bgruening John said "I will update the slides for all your inline comments as I make my way back through the slides to synchronize them with the updated Planemo docs.", so I'll leave them to him.